Split getting started into SGX and simulation mode (#72)
edgelesssys/marblerun.sh: content/docs/getting-started/quickstart-simulation.md

```

Create and annotate the emojivoto namespace for [auto-injection]({{< ref "docs/features/auto-injection.md" >}}):

```bash
kubectl create namespace emojivoto
marblerun namespace add emojivoto --no-sgx-injection
```

Using the `--no-sgx-injection` flag we tell Marblerun to not inject configurations specific to SGX into starting pods.

### Step 2.2: Deploy emojivoto

Finally, install the demo application onto your cluster.
Please make sure you have [Helm](https://helm.sh/docs/intro/install/) ("the package manager for Kubernetes") installed at least at Version v3.2.0.
Install emojivoto into the emojivoto namespace by running:

```bash
helm install -f ./kubernetes/nosgx_values.yaml emojivoto ./kubernetes --create-namespace -n emojivoto
```

## Step 3: Watch it run

You can now check the Marblerun log and see the services being authenticated by the Coordinator.

```bash
kubectl -n marblerun logs -f -l edgeless.systems/control-plane-component=coordinator
```

Port forward the front-end web service to access it on your local machine by running:

```bash
sudo kubectl -n emojivoto port-forward svc/web-svc 443:443 --address 0.0.0.0
```

Now visit [https://localhost](https://localhost).

> You'll be presented with a certificate warning because your browser does not know Marblerun's root certificate as a root of trust.

Reviewer: ```suggestion
You'll be presented with a certificate warning because your browser does not know trust Marblerun's self-signed root certificate by default.
```

Author: We do not install the Marblerun root cert in the browser during the quickstart, so it should be correct to assume the browser does not know the certificate at all.
The certificate that we dont trust is the one supplied by the web service frontend (which is signed by Marbleruns root cert)

Reviewer: I mainly rewrote this just to get rid of having "root" twice in a sentence.
